cmd/osquery-perf/mac10.14.6.tmpl

```diff
@@ -217,6 +217,21 @@
   }
 ]
 {{- end }}
+{{ define "fleet_detail_query_os_version_unix_like" -}}
+[
+  {
+    "name":"Mac OS X",
+    "version":"10.14.6",
+    "major":"10",
+    "minor":"14",
+    "patch":"6",
+    "build":"18G3020",
+    "platform":"darwin",
+    "platform_like":"darwin",
+    "codename":""
```

Comment:
why codename is empty in all responses?

Reply:
This is one of the fields that macOS hosts do not report (in contrast to other operating systems).

Reply:
gotcha, but there are new queries for other operating systems below and none contain codename, not saying is wrong, but I want to double check

Reply:
Good point! I'm not sure why it wasn't included in those Ubuntu templates originally. My guess is because it wasn't relevant to the vulnerabilities test cases. I'll go back over those templates and make sure the fields are all tracking real-world data.

As an aside, it looks like we haven't been collecting any codename data at all! Our column was named `code_name` but in osquery it is `codename` so it was never getting ingested. Thanks for asking questions :)